{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -1,30 +1,49 @@
-[tool.poetry]
+[build-system]
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
+build-backend = "poetry_dynamic_versioning.backend"
+
+[project]
 name = "{{cookiecutter.__project_slug}}"
-version = "0.1.0"
 description = "{{cookiecutter.project_description}}"
 authors = ["{{cookiecutter.__author}}"]
 license = "{{cookiecutter.license}}"
 readme = "README.md"
 include = ["README.md", "src/{{cookiecutter.__project_slug}}/schema", "project"]
 
+requires-python = ">=3.9"
+
+dynamic = ["version"]
+
+dependencies = [
+  "linkml-runtime >=1.8.0",
+]
+
+[project.optional-dependencies]
+docs = [
+  "mkdocs-material >=8.2.8",
+  "mkdocs-mermaid2-plugin >=1.1.1",
+  "schemasheets >= 0.1.14",
+]
+dev = [
+  # "pid4cat_model[docs]",  # no support for recursive optional dependencies: poetry#3369
```

Comment:
Can this be removed? Was it just a bit of errant copy-pasting?

Reply:
Yeah I can remove it. Typically you can import another group if there wasn't this poetry issue (see https://hynek.me/articles/python-recursive-optional-dependencies/).